tests/hazmat/backends/test_openssl_memleak.py

```diff
@@ -113,18 +129,22 @@ def free(ptr, path, line):
         )
         assert result == 1
 
-    remaining = set(heap) - start_heap
+    remaining = set(heap) - set(start_heap)
 
-    if remaining:
-        sys.stdout.write(json.dumps(dict(
+    # The constant here is the number of additional bytes of memory
+    # consumption that are allowed in reallocs of start_heap memory.
+    if remaining or start_heap_realloc_delta[0] > 1024:
```

Comment:
Can the threshold be set lower?

Reply:
In my conda-forge test, it turns out that there were a couple that realloc'ed 192 additional bytes. So it could drop to 256, say, but not quite as small as I originally thought.

Reply:
256 sounds good to me. thanks.

Reply:
Actually it looks like some of the Ubuntu tests actually need an even higher number:

```
AssertionError: {'start_heap_realloc_delta': 768}
```

I think what must be happening here is that on Ubuntu the `realloc()` call is returning the same pointer as before, which is the only way I can think of that a `realloc()` wouldn't have triggered issues before. OK if I raise the limit back to 1024?

Reply:
I've gone ahead and returned the setting to 1024.

Reply:
Oh fascinating, so it's not that this hits a different code path on linux than Windows, it's that on Windows the allocator patterns are different.

I guess this is fine.